lib/atomic-management.js

```diff
@@ -358,5 +359,31 @@ export default {
       CSON.readFileSync(atom.config.getUserConfigPath()),
       atom.config.getUserConfigPath()
     );
+  },
+
+  /**
+   * Installs the packages specified using apm install. Will silently ignore
+   * any package names that contain spaces, because that is invalid.
+   * 
+   * @param {Array<String>} packageNames The package names to install
+   */
+  installPackages(packageNames) {
```

Comment:
This function isn't actually called anywhere. Did you mean to hook this up to a commands (see line 42) that gets called from the notification

Reply:
Right, I just created the function, which I'll integrate with the notifications at some later point (in other words, not tonight). Wanted to wait until the other pull requests were integrated first, because there were a lot of changes being made and I wanted the dust to clear. But sent this pull request just in case you two wanted to take a look at how I plan to integrate the "install packages for the user" functionality.

Reply:
oh gotcha